internal/http/services/owncloud/ocdav/put.go

```diff
@@ -168,6 +168,20 @@ func (s *svc) handlePut(ctx context.Context, w http.ResponseWriter, r *http.Requ
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+		if mtime := r.Header.Get(net.HeaderOCMtime); mtime != "" {
+			smRes, err := client.SetArbitraryMetadata(ctx, &provider.SetArbitraryMetadataRequest{
```

Comment:
This means we need to do another call to the storage provider which is not nice. Can we instead send the `mtime` with the `TouchFileRequest`? I'm thinking about putting it in the opaque map, then extract it in the storage provider and pass it to decomposedfs. Similar like we do with the `markprocessing` variable. 

See https://github.com/cs3org/reva/blob/edge/pkg/storage/utils/decomposedfs/decomposedfs.go#L619 and https://github.com/cs3org/reva/blob/edge/internal/grpc/services/storageprovider/storageprovider.go#L671

We would need to change the interface but I think that is better than the extra call